test/skimNtuple_HHbtag.cpp

```diff
@@ -2904,11 +3005,34 @@ int main (int argc, char** argv)
 		  idFakeSF_mutauFR_eta0p8to1p2_down = except_vsMu * idSF_leg2_deep_vsMu_down[2];
 		  idFakeSF_mutauFR_eta1p2to1p7_down = except_vsMu * idSF_leg2_deep_vsMu_down[3];
 		  idFakeSF_mutauFR_etaGt1p7_down    = except_vsMu * idSF_leg2_deep_vsMu_down[4];
-	
+
 		  idFakeSF_etauFR_barrel_up		= except_vsEle * idSF_leg2_deep_vsEle_up[0];
 		  idFakeSF_etauFR_endcap_up		= except_vsEle * idSF_leg2_deep_vsEle_up[1];  
 		  idFakeSF_etauFR_barrel_down	= except_vsEle * idSF_leg2_deep_vsEle_down[0];
 		  idFakeSF_etauFR_endcap_down	= except_vsEle * idSF_leg2_deep_vsEle_down[1];
+
+		  if(pType==0){
+			idFakeSF_muID_up = except_leg1 * idSF_leg1_muID_up;
+			idFakeSF_muID_down = except_leg1 * idSF_leg1_muID_down;
+			idFakeSF_muIso_up = except_leg1 * idSF_leg1_muIso_up;
+			idFakeSF_muIso_down = except_leg1 * idSF_leg1_muIso_down;
```

Comment:
Aren't you consistently missing one term?
For instance, shouldn't the first be:

```
idFakeSF_muID_up = except_leg1 * idSF_leg1_muID_up * <nominal Iso term>;
```

?

Reply:
`idSF_leg1_muID_up` already includes the nominal iso SF